### PR TITLE
Fix Formatting Errors

### PR DESCRIPTION
**Pull request by gitr0n1n**
Fix Formatting Errors on author fields

### CODE REVIEW

**Comment by bobsummerwill** [comment]
/raw PONG _GHMoaCXLT is how the author chose to identify for two of these ECIPs, and there is no need to DOX the author with another name.   Also, the name used in this PR is not a full name, either.

Best NOT to change this.

File: _specs/ecip-1077.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 ecip: 1077
 title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
-author: Mr. Meows D. Bits
+author: Mr. Meows D. Bits (@meowsbits)
```
Comment (meowsbits) [inline _specs/ecip-1077.md:4]:
```suggestion
author: Mr. Meows D. Bits
```

**Review by meowsbits — CHANGES_REQUESTED** [review]
Why are parenthetical identities better than bracketed? 

EDIT
Oh, I see, you just invented this... https://github.com/ethereumclassic/ECIPs/blame/master/_specs/ecip-1000.md#L158

Maybe we could consider some kind of syntax for specifying the app-handle relationship, like

```
John Doe <johndoe123@hotmail.com>
Jonh Doe (github:johndoe)
```

The problem is that `@johndoe` is ambiguous/useless for the repo as a decentralizable collection of documents, since it presumes the existence of and integration with the Github corporation.

**Comment by gitr0n1n** [comment]
> /raw PONG _GHMoaCXLT is how the author chose to identify for two of these ECIPs, and there is no need to DOX the author with another name. Also, the name used in this PR is not a full name, either.
> 
> Best NOT to change this.

This comment doesn't make much sense:
https://github.com/q9f
http://ecips.ethereumclassic.org/ECIPs/ecip-1103

File: _specs/ecip-1077.md (hunk 1 above)
Comment (gitr0n1n) [reply _specs/ecip-1077.md:4]: Just following the ECIP-1000

"The Author header lists the names and email addresses/GitHub handles of all the authors/owners of the ECIP. The format of the Author header value must be:

Random J. User <123@dom.ain>
Random J. User (@random.j.user)
If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions."

Comment (gitr0n1n) [reply _specs/ecip-1077.md:4]: Just following the ECIP-1000

The Author header lists the names and email addresses/GitHub handles of all the authors/owners of the ECIP. The format of the Author header value must be:

Random J. User <123@dom.ain>
Random J. User (@random.j.user)
If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions.

**Comment by gitr0n1n** [comment]
> Why are parenthetical identities better than bracketed?
> 
> EDIT Oh, I see, you just invented this... https://github.com/ethereumclassic/ECIPs/blame/master/_specs/ecip-1000.md#L158
> 
> Maybe we could consider some kind of syntax for specifying the app-handle relationship, like
> 
> ```
> John Doe <johndoe123@hotmail.com>
> Jonh Doe (github:johndoe)
> ```
> 
> The problem is that `@johndoe` is ambiguous/useless for the repo as a decentralizable collection of documents, since it presumes the existence of and integration with the Github corporation.

Just fixing broken links. Absent links. 👍

**Comment by meowsbits** [comment]
Generally, editing other people signatures is going to feel wrong to me in 99% of the cases. We have no hard requirements for identitiy/entity assertion, nor should we.

**Comment by gitr0n1n** [comment]
> Generally, editing other people signatures is going to feel wrong to me in 99% of the cases. We have no hard requirements for identitiy/entity assertion, nor should we.

These are not "signatures" these are ECIP `author` headers in the Preamble.

File: _specs/ecip-1077.md (hunk 1 above)
Comment (meowsbits) [reply _specs/ecip-1077.md:4]: I have a different interpretation: It does not mean than an author MUST register an email (OR a Github handle, now); it means that IF an author registers one of those values, they should be formatted as described.

**Comment by gitr0n1n** [comment]
> Generally, editing other people signatures is going to feel wrong to me in 99% of the cases. We have no hard requirements for identitiy/entity assertion, nor should we.
> 
> > Just fixing broken links.
> 
> Where are they broken? Can you please describe the issue better?

Sure. Here is  an example of a broken link.

image

**Comment by meowsbits** [comment]
That isn't a broken link because it isn't a link. Its plain text.

**Comment by gitr0n1n** [comment]
> That isn't a broken link because it isn't a link. Its plain text.

Here is another for you.
image

**Comment by meowsbits** [comment]
I am OK with punctuation-only edits. But changing, adding, or removing people's names is substantive, and not OK with me.

**Comment by gitr0n1n** [comment]
> I am OK with metadata-syntax/punctuation-only edits. But changing, adding, or removing people's names (real or claimed) is substantive, and not OK with me.

Great we will start with that as I comb through this repo for generally immaterial admin clean up.

**Comment by IstoraMandiri** [comment]
> That isn't a broken link because it isn't a link. Its plain text.

In this case it looks like the author wasn't was accidentally using the wrong syntax `(` instead of `<`, which is why the link is broken, see https://github.com/ethereumclassic/ECIPs/pull/479#issuecomment-1053909982

**Comment by gitr0n1n** [comment]
> I agree, would you support overriding current `(@blah)` with this new `(github:blah)` or `(twitter:blah)`, and while we are at it should we replace `<hi@joe.com>` with `(email:hi@joe.com)` ?
> 
> This is clearly a confusing feature currently

I like this formatting you are suggesting because it is clearly readable in markdown text.
e.g.:  `(github:blah)` or `(twitter:blah)

**Comment by meowsbits** [comment]
I don't opinions on the precise handle/tag syntax... Probably there are prior arts and patterns that we can reuse instead of reinventing... 

Also. just by the way, `ecip_validator` is a gem that codifies these kinds of validations, and is currently run in the Github Actions workflow. That may be a good place for some of these kinds of validations we're talking about here.

**Comment by gitr0n1n** [comment]
So what's going on with this PR @ethereumclassic/ecip-editors ? We good to push and merge?

**Review by meowsbits — APPROVED** [review]

**Pull request merged**